lib/APINotes/APINotesWriter.cpp

```diff
@@ -584,8 +584,12 @@ namespace {
       emitCommonTypeInfo(out, info);
 
       uint8_t payload = 0;
+      if (auto swiftImportAsNonGeneric = info.getSwiftImportAsNonGeneric()) {
+        payload |= (0x01 << 1) | swiftImportAsNonGeneric.getValue();
+      }
+      payload <<= 3;
```

Comment:
This is making room for the *next* chunk of data, not moving out the *previous* data.